### ISSUE TITLE
[BITCOIN DEV KIT] MARKETER

### ISSUE THREAD

**Issue by aassoiants** [Bitcoin-Product-Community/opportunities#7]
# Project Details
Project name: Bitcoin Development Kit
Project summary (~5 sentences): BDK trying to make it easy for anyone to create a mobile wallet. Library + language bindings. On chain only. Bitcoin wallets.
Is it open-source?: Yes
Role: Marketer

# Team Dynamics
About how many core contributors are involved? What are their roles?
- 7, mostly devs

What are the communication mediums? Discord, signal, github projects
What stage is the project in? Pre 1.0
How long has it been running for? >3 years
What are the goals of the project? Marketing
Do you have any regular meetings? Yes 
What timezone are your contributors in? TBD
What the project needs in general? Marketing for 1.0 release

# About the Role
What specific role(s) or skill(s) are you looking for? 
- Marketing. Ideally is bitcoin-only.
- List of projects and ask them what would incentivize the to use BDK. Collect feedback from users and feeding it back to their roadmap. For examples if they don’t want or can’t use BDK, get why. 
- Domain name, website.
- Setting workshops at conferences, speaking engagements.

How many hours a week minimum will they need to commit and for how long? At least 3 hours a week for ideally minimum 2 years
Is this a paid position? No
What can the contributor expect when onboarding to your project? Onboarding call, gradual ramp-up

# How to Connect
If a contributor is interested, how should they get involved? DM

**Comment by nillawafa** [comment]
I've communicated with Art by DM to express interest in this role 👍.

A little bit about my background: I’m not a professional marketer, I’ve just been trying to find a way to start contributing to the space beyond lurking on twitter and going to meetups. My fiat job is as an analyst and I started looking into Bitcoin seriously in 2020. Bitcoin and FOSS I just think are a big part of our future. A goal of mine with this would be to learn a bit about marketing, FOSS, and startups. Hopefully that’s acceptable
